lib/iris/tests/integration/test_netcdf.py

```diff
@@ -210,5 +210,28 @@ def test_lazy_mask_preserve_fill_value(self):
             self.assertCDL(other_filename)
 
 
+class TestCellMeasures(tests.IrisTest):
+    def setUp(self):
+        self.fname = tests.get_data_path(('NetCDF', 'ORCA2', 'votemper.nc'))
+
+    def test_load_raw(self):
```

Comment:
I don't see the value of testing 'load_raw', 'load' and 'load_cube' separately here, as this just seems to be testing two things at once.
In fact, I think by design you can not have any merging when loading netcdf data from a *single* file, so there is no different behaviour to be expected from using "load_raw".  Neither is there any new behaviour there, specific to cell measures.
So I think I'd prefer this to just use "load_cube" throughout.